2017-bae-europe/eu_future/buff_but_sink.xml

```diff
@@ -0,0 +1,5827 @@
+
```

Comment:
Oh, this is just a test input file I made, that somehow got committed. Please disregard it. I apologize.